src/checkpoint/checkpoint_txn.c

```diff
@@ -1843,8 +1843,7 @@ __checkpoint_lock_dirty_tree_int(WT_SESSION_IMPL *session, bool is_checkpoint, b
          * of WiredTiger checkpoints dropped per checkpoint.
          */
         if (is_wt_ckpt)
-#define WT_MAX_CHECKPOINT_DROP 4
-            if (++max_ckpt_drop >= WT_MAX_CHECKPOINT_DROP)
+            if (++max_ckpt_drop >= WTI_MAX_CHECKPOINT_DROP)
```

Comment:
From the slack discussion I though the intention was to keep the #define here.

Reply:
I'm sorry, I added the prefix and moved it as well.

Reply:
fair enough, just checking

Reply:
38887782e37d5c31aff8b624cf050aaffc0385a1